deploy/kustomize/patches/clusterserviceversion-patch.yaml

```diff
@@ -0,0 +1,169 @@
+#
+# Operator display name
+#
+- op: replace
```

Comment:
Is this perhaps shorter as a strategic merge patch?

Reply:
Do you mean to split the patch into smaller patches and use [`patchesStrategicMerge`](https://kubectl.docs.kubernetes.io/pages/reference/kustomize.html#patchesstrategicmerge) as? I could only find good examples of how to use kustomize for strategic merge patch.

Reply:
It would be a single file which contains a single object which has just the fields that we want to replace. I'm currently trying this out, stay tuned...

Reply:
As I suspected, the strategic merge patch is easier to read, see PR https://github.com/avalluri/pmem-CSI/pull/2. Feel free to squash that into your commit.

I verified that the output is the same, with one exception: spec/provider/url wasn't set before. I added "https://intel.com" to get rid of the placeholder "https://your.domain".

Reply:
I took your patch and made changes to use the drafted CSV as the base manifest for `operator-sdk`. So that we needed not to patch explicitly the generated catalog except the version-specific links.